src/app/shared/view-mode-switch/view-mode-switch.component.html

```diff
@@ -15,6 +15,6 @@
        routerLinkActive="active"
        [class.active]="currentMode !== viewModeEnum.List"
        class="btn btn-secondary">
-          <i class="fas fa-th-large" title="{{'search.view-switch.show-grid' | translate}}"></i>
+          <i class="fa fa-th-large" title="{{'Show as grid' | translate}}"></i>
```

Comment:
And another change from `fas` to `fa`.

Reply:
Unclear how that replace happened, reverted today.